x/name/keeper/keeper.go

```diff
@@ -291,3 +294,34 @@ func isValidUUID(s string) bool {
 	}
 	return true
 }
+
+func (keeper Keeper) addRecord(ctx sdk.Context, name string, addr sdk.AccAddress, restrict, isModifiable bool) error {
+	key, err := types.GetNameKeyPrefix(name)
```

Comment:
I noticed that we still have amino in this module (albeit as a deprecated legacy compatibility path) ... this would be a good time to clean that up with a module upgrade method.  I am not sure how much longer amino will be present at all.

Reply:
I created a separate issue for this since the removal of legacy amino code will affect the attribute module as well.